event_registration_cancel_reason/wizard/event_registration_cancel_log_reason.py

```diff
@@ -23,12 +23,13 @@ def default_get(self, var_fields):
             var_fields)
         registrations = self.env['event.registration'].browse(
             self.env.context['active_ids'])
-        event_type = registrations.mapped("event_id.type")
-        if len(event_type) != 1:
-            raise exceptions.ValidationError(
-                _("You cannot cancel registrations from events of different "
-                  "types at once."))
-        res['event_type_id'] = event_type.id
+        first_type = registrations[0].event_id.type
```

Comment:
I still cannot find a real use case where you could possibly get to this wizard and `registrations` would be empty (and less where you'd wish the process not to get interrupted). But nevermind, let me change it.